Week1_final.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>Here would be a good place to isolate the equations from the text. You could also consider making this text smaller than the learning goals text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-wildbillcolorado/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>